activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -601,6 +601,7 @@ def foreign_keys(table_name)
 
             options[:on_delete] = extract_foreign_key_action(row["on_delete"])
             options[:on_update] = extract_foreign_key_action(row["on_update"])
+            options[:valid] = row["valid"]
```

Comment:
That doesn't make much sense to me. At this point, the foreign key has already been created. I'd expect a `valid?` or `validated?` method to describe the status of an existing foreign key and `options[:validate]` to be used only during creation.

Either way, I'll make the change in a subsequent commit.